sixtyfps_runtime/rendering_backends/gl/graphics_window.rs

```diff
@@ -466,10 +467,20 @@ impl PlatformWindow for GraphicsWindow {
             GraphicsWindowBackendState::Unmapped => {}
             GraphicsWindowBackendState::Mapped(window) => {
                 let title = window_item.title();
+                let icon = window_item.icon();
                 let mut size: LogicalSize<f64> = {
                     let backend = window.backend.borrow();
                     let winit_window = backend.window();
                     winit_window.set_title(&title);
+                    if let Some(img) = crate::images::CachedImage::new_from_resource((&icon).into())
+                        .and_then(|i| i.into_image())
```

Comment:
This would always load the image from disk when showing the window. Maybe this could use the cache that we have (untested!):

```suggestion
                    crate::IMAGE_CACHE.with(|image_cache| {
                        image_cache
                        .borrow_mut()
                        .load_image_resource(&icon).into())
                    }).and_then(|i| i.into_image())
```

Reply:
the problem is that the image cache turns it into a femtovg texture, does it not?  What i really want is a RGB thing. 
It only reloads it when the title or the width or height becomes dirty

Reply:
`crate::IMAGE_CACHE` will only create decoded images on the CPU side. femtovg textures require a window, GL context, etc., which is why the `GraphicsWindow` has a `texture_cache` member.